AccountService.js

```diff
@@ -40,4 +40,71 @@ export class AccountService {
     });
     return response.data;
   }
+
+  /**
+   * @method getAll
+   * @param {Object} config
+   * @param {string} config.url - the base url
+   * @param {string} config.email - the user's email
+   * @param {string} config.after - an account's DID
+   * @param {number} config.limit - how many accounts to return
+   * @return {Array} data
+   * @description calls on the index route and returns all
+   * accounts that match the email passed in.
+   */
+  async getAll({url = this.config.urls.base, email, after, limit}) {
+    const response = await axios.get(url, {
+      params: {email, after, limit},
+      headers: {'Accept': 'application/ld+json, application/json'}});
```

Comment:
Are the default axios headers not working for some reason?

Reply:
If there is a requirement to specify headers, they should be set to some variable and not be string literals throughout.

Reply:
just copied the existing pattern here.

Reply:
@dlongley @gannan08 is there a reason to pass headers into axios?  Apparently that practice got started somewhere.

Reply:
ok created a const headers and then use it in all the requests